web/tests/choice-params-with-default.spec.ts

```diff
@@ -8,7 +8,7 @@ test.beforeAll(async ({ browser }) => {
   await workspace.addBox({ id: 'ex0', name: 'Create example graph', x: 100, y: 100 });
 });
 
-test('test pagerank default choice values', async () => {
+test('pagerank default choice values', async () => {
```

Comment:
The original strings are state names, and even as state names they are not very clear. I'm thinking how we could improve them. Here I'm trying to not change much, but at least make the name specify what we are testing. We're testing these default choice values. Then we are testing editing them. It makes a bit more sense in my mind.